### PR TITLE
Components

### PR DESCRIPTION
**Pull request by hzik**
### Motivation

Kentico Cloud has linked items element as well as linked items as a part of the rich text. Then there are components as a part of the rich text, but there are components missing as a stand-alone element so editors can create as many components as they need with no related items created.

### How to test

No special testing needed - just follow README instructions?

### CODE REVIEW

**Comment by hzik** [comment]
done :)

File: Components/components.html [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+	
+	function setup(value, name, inputs) {	
+		objectName = name;
+		$('#add_object').text("Add "+objectName);
+		
+		if (checkUniqueNames(inputs)) {
+			mainObject = inputs;
+		}
+		else {
+			showError('Your input configuration contains suplicated names. Please name your inputs uniquely.');
```
Comment (Simply007) [inline Components/components.html]: Typo:

```suggestion
			showError('Your input configuration contains duplicated names. Please name your inputs uniquely.');  
```

Comment (hzik) [reply Components/components.html]: I see, I'll add it :)

**Comment by hzik** [comment]
No configuration error message added :)

**Pull request merged**